test-skripsi/test-prod.tsx

```diff
@@ -10,27 +17,33 @@
 
 	constructor() {
 		this.inventory = [];
+		this.loadInventory();
+	}
+
+	private findProductById(productId: number): Product | undefined {
+		return this.inventory.find((p) => p.id === productId);
 	}
```

Comment:
Good use of a private method to encapsulate the logic for finding a product by ID. This improves code reusability and maintainability.